apps/test/netsim/NetSimTable.js

```diff
@@ -186,7 +188,7 @@ describe("NetSimTable", function () {
     assertEqual(apiTable.log(), '');
 
     // Until poll interval has passed.
-    netsimTable.lastFullUpdateTime_ = Date.now() - (netsimTable.pollingInterval_ + 1);
+    netsimTable.lastRefreshTime_ = Date.now() - (netsimTable.pollingInterval_ + 1);
```

Comment:
The current behavior is a little surprising.  With a 1 second polling interval, I would expect that if I wait 1 second after a poll that I would poll again, which is not the case.